mm.c

```diff
@@ -102,6 +102,12 @@ team_t team = {
 #define GET_PRED(bp) (*(GET_PTR_PRED_FIELD(bp)))
 #define GET_SUCC(bp) (*(GET_PTR_SUCC_FIELD(bp)))
 
+// Given pointer to current block, return pointer to header of previous block
+#define PREV_BLOCK_IN_HEAP(header_p) ((char **)(header_p) - GET_SIZE((char **)(header_p) - 1) - HDR_FTR_SIZE)
```

Comment:
can we use the macro GET_TOTAL_SIZE to get the total size of the previous block so that it's clearer ?